Source/Fuse.Navigation/RoutePageProxy.uno

```diff
@@ -213,7 +213,13 @@ namespace Fuse.Navigation
 			float p = 1; //defensive value for odd state
 			for (int i=0; i < _levels.Count; ++i)
 			{
-				var lp = _levels[i].PageProxy.Navigation.GetPageState(_levels[i].PageProxy.Page).Progress;
+				var level = _levels[i];
```

Comment:
Since you only ever touch `level.PageProxy`, maybe store that in a local variable instead?